app/experimenter/experiments/tests/api/v3/test_serializers.py

```diff
@@ -122,13 +122,17 @@ def test_serializer_creates_changelog_for_updates(self):
             name="rapid experiment",
             slug="rapid-experiment",
             objectives="gotta go fast",
+            audience="AUDIENCE 1",
+            features=["FEATURE 1"],
             public_description=Experiment.BUGZILLA_RAPID_EXPERIMENT_TEMPLATE,
         )
 
         self.assertEqual(experiment.changes.count(), 1)
         data = {
             "name": "changing the name",
             "objectives": "changing objectives",
+            "audience": "AUDIENCE 1",
+            "features": ["FEATURE 1"],
```

Comment:
I'd like to see some failure tests for like

- invalid audience choice
- missing audience choice
- invalid feature choices
- empty feature choices

Reply:
will do